modin/pandas/test/test_io.py

```diff
@@ -259,28 +259,36 @@ def _make_parquet_dir(
 @pytest.mark.filterwarnings(default_to_pandas_ignore_string)
 class TestCsv:
     # delimiter tests
-    @pytest.mark.parametrize("sep", [None, "_", ",", ".", "\n"])
-    @pytest.mark.parametrize("delimiter", ["_", ",", ".", "\n"])
+    @pytest.mark.parametrize("sep", ["_", ",", "."])
     @pytest.mark.parametrize("decimal", [".", "_"])
     @pytest.mark.parametrize("thousands", [None, ",", "_", " "])
-    def test_read_csv_delimiters(
-        self, make_csv_file, sep, delimiter, decimal, thousands
-    ):
+    def test_read_csv_delimiters(self, make_csv_file, sep, decimal, thousands):
```

Comment:
```suggestion
    def test_read_csv_seps(self, make_csv_file, sep, decimal, thousands):
```
Should we rename this test for consistency with the parameter? or just rename the parameter name `sep` to `delimeter`.

Reply:
renamed